api-tester/src/main/java/com/revenuecat/apitester/java/DeprecatedPurchasesAPI.java

```diff
@@ -15,8 +24,35 @@ static void check(final Purchases purchases,
                       final SkuDetails skuDetails,
                       final StoreProduct storeProduct,
                       final Package packageToPurchase,
+                      final SubscriptionOption subscriptionOption,
                       final UpgradeInfo upgradeInfo) {
+        final ProductChangeCallback purchaseChangeListener = new ProductChangeCallback() {
+            @Override
+            public void onCompleted(@Nullable StoreTransaction storeTransaction, @NonNull CustomerInfo customerInfo) {
+            }
+
+            @Override
+            public void onError(@NonNull PurchasesError error, boolean userCancelled) {
+            }
+        };
+        final PurchaseCallback makePurchaseListener = new PurchaseCallback() {
+            @Override
+            public void onCompleted(@NonNull StoreTransaction storeTransaction, @NonNull CustomerInfo customerInfo) {
+            }
+
+            @Override
+            public void onError(@NonNull PurchasesError error, boolean userCancelled) {
+            }
+        };
```

Comment:
anyone have thoughts on how to migrate this? can we just change one of the parameters to be nullable and call it out, and keep the same name?

Reply:
@vegaro @joshdholtz ^^